onnxruntime/core/framework/provider_bridge_ort.cc

```diff
@@ -749,6 +753,14 @@ struct ProviderHostImpl : ProviderHost {
   int32_t Tensor__GetElementType(const Tensor* p) override { return p->GetElementType(); }
   MLDataType Tensor__DataType(const Tensor* p) override { return p->DataType(); }
 
+  // TensorSeq
```

Comment:
// TensorSeq (wrapped)
(since you're wrapping the type vs directly resolving externals)

Reply:
Done.